nhsd-ord-api/role-ids.md

```diff
@@ -1,5 +1,7 @@
 # Role IDs
 
+This is a list of the role IDs returned by the NHS ORD API.
```

Comment:
I think we could also add a line about how people can get this data for themselves.

Reply:
> I think we could also add a line about how people can get this data for themselves.

No problem, I have done this if you would be able to review?